### PR TITLE
Organizing add section in docs

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add section about organizing assets.
- add floors
- add areas
- Definitions for floors, area, label, category in organizing overview page
- related topics on labels, categories, and how to work with them will be added in separate PRs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration others are adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/110741
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/organizing.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Organizing your assets"
+description: "Organizing and grouping your assets makes it easier to find them and allows you target groups in automations."
```
Comment (frenck) [inline source/_docs/organizing.markdown]:
```suggestion
description: "Organizing and grouping your assets makes it easier to find them and allows you to target groups in automations."
```

File: source/_docs/organizing.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+Once you have more devices, you may want to target entire groups of devices in automations. It also becomes more challenging to find items in lists. There are a few tools to organize your assets: [Areas](/docs/organzing/#areas/), [floors](/docs/organzing/#floors/), [labels](/docs/organzing/#labels/), and [categories](/docs/organzing/#categories/).
```
Comment (frenck) [inline source/_docs/organizing.markdown]:
```suggestion
Once you have more devices, you may want to target entire groups of devices in automations. It also becomes more challenging to find items in lists. There are a few tools to organize your assets: [Areas](#areas), [floors](#floors), [labels](#labels), and [categories](#categories).
```

File: source/_docs/organizing.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Areas
+
+An area in Home Assistant is a logical grouping of {% term devices %} and {% term entities %} that are meant to match areas (or rooms) in the physical world of your home. For example, the `living room` area groups devices and entities in your living room. Areas allow you to target an entire group of devices with a service call. For example, turning off all the lights in the living room.
+Areas can be assigned to floors. Areas can also be used to automatically generate cards, such as the [Area card](/dashboards/area/).
```
Comment (frenck) [inline source/_docs/organizing.markdown]:
```suggestion
An area in Home Assistant is a logical grouping of {% term devices %} and {% term entities %} that are meant to match areas (or rooms) in the physical world of your home. For example, the "Living room" area groups devices and entities in your living room. Areas allow you to target an entire group of devices with a service call. For example, turning off all the lights in the living room.
Areas can be assigned to floors. Areas can also be used to automatically generate cards, such as the [Area card](/dashboards/area/).
```

File: source/_docs/organizing/areas.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Areas"
+description: "Group your devices and entities in areas and group areas in floors."
+---
+
+An area in Home Assistant is a logical grouping of {% term devices %} and {% term entities %} that are meant to match areas (or rooms) in the physical world of your home. For example, the `living room` area groups devices and entities in your living room. Areas allow you to target an entire group of devices with a service call. For example, turning off all the lights in the living room.
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown]:
```suggestion
An area in Home Assistant is a logical grouping of {% term devices %} and {% term entities %} that are meant to match areas (or rooms) in the physical world of your home. For example, the "Living room" area groups devices and entities in your living room. Areas allow you to target an entire group of devices with a service call. For example, turning off all the lights in the living room.
```

File: source/_docs/organizing/areas.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Areas can be assigned to floors. Areas can also be used to automatically generate cards, such as the [Area card](/dashboards/area/).
+
+## Creating a area
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown]:
```suggestion
## Creating an area
```

File: source/_docs/organizing/areas.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+Follow these steps to create a new area from the **Areas** view.
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown:11]:
```suggestion
Follow these steps to create an new area from the **Areas** view.
```

File: source/_docs/organizing/areas.markdown [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %} and select **Create area**.
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown]:
```suggestion
1. Go to {% my areas title="**Settings** > **Areas, labels & zones**" %} and select **Create area**.
```

File: source/_docs/organizing/areas.markdown [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+2. In the dialog, enter the area details:
+   - Give the area a **Name** (required).
+   - Add an icon (We use [Material icons](https://pictogrammers.com/library/mdi/)).
+   - Assign the area to a floor.
+     - If you have not created floors yet, you can create a new one.
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown]: Maybe link this to the floors procedure?

File: source/_docs/organizing/areas.markdown [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    ![Create area dialog](/images/organizing/create_area_01.png)
+3. Select **Add**.
+
+   **Result**: A new area is created.
+
+## Assigning areas to floors and add labels
+
+If an area has not yet been assigned to a floor, it is shown in the **Unassigned areas** section. Follow these steps to assign an area to a floor.
+
+1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %} and select **Create area**.
```
Comment (frenck) [inline source/_docs/organizing/areas.markdown]:
```suggestion
1. Go to {% my areas title="**Settings** > **Areas, labels & zones**" %} and select **Create area**.
```

File: source/_docs/organizing/floors.markdown [hunk 10]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Floors"
+description: "Group your areas per floor"
+---
+
+A floor in Home Assistant is a logical grouping of areas that are meant to match the physical floors in your home. Devices and entities
+cannot be assigned to floors directly but to areas. Floors can be used in automations and scripts as a target for actions. For example, to turn off all the lights on the downstairs floor when you go to bed.
+
```
Comment (frenck) [inline source/_docs/organizing/floors.markdown]:
```suggestion
A floor in Home Assistant is a logical grouping of areas meant to match your home's physical floors. Devices and entities
cannot be assigned to floors directly but to areas. Floors can be used in automations and scripts as a target for actions. For example, to turn off all the lights on the downstairs floor when you go to bed.

```

File: source/_docs/organizing/floors.markdown [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+## Creating a floor
+
+Follow these steps to create a new floor.
+
+1. Go to {% my zones title="**Settings** > **Areas, labels & zones**" %} and select **Create floor**.
```
Comment (frenck) [inline source/_docs/organizing/floors.markdown]:
```suggestion
1. Go to {% my areas title="**Settings** > **Areas, labels & zones**" %} and select **Create floor**.
```

File: source/_includes/asides/docs_navigation.html [hunk 12]
```diff
@@ -124,11 +124,12 @@ <h1 class="title delta">Topics</h1>
...
         <ul>
-          <li>
-            {% active_link /docs/frontend/browsers/ Browser compatibility list %}
-          </li>
           <li>{% active_link /dashboards Dashboards %}</li>
+          <li>{% active_link /docs/organizing/ Organizing %}</li>
           <li>{% active_link /docs/frontend/icons/ Icons%}</li>
+          <li>
+            {% active_link /docs/frontend/browsers/ Browser compatibility list%}
+          </li>
```
Comment (frenck) [inline source/_includes/asides/docs_navigation.html:132]: Not for this PR, just a note: But I think we should consider ditching this (if we want a replacement, a generic "system requirements page" might be a better fit).

**Review by agners — APPROVED** [review]
On a quick glance, LGTM.

**Comment by frenck** [comment]
Thanks for this one @c0ffeeca7 👍

**Pull request merged**